examples/todomvc/src/models/todos.js

```diff
@@ -0,0 +1,48 @@
+import { types, destroy } from 'mobx-state-tree'
+
+const Todo = types.model(
+  {
+    text: 'Learn Redux',
+    completed: false,
```

Comment:
I noticed that my vscode is switching between single quote and double quote... maybe we should define it in the settings.json as well?